frontend/src/CSets.js

```diff
@@ -223,114 +298,82 @@ function ConceptSetsPage(props) {
-    // (data && <div style={{
-    (<div style={{
-        padding: '1px 15px 1px 15px',
-        margin: '5px 5px 5px 5px',
-        border: '5px 5px 5px 5px',
-        background: '#d3d3d3',
-        'border-radius': '10px',
-      }}>
-        <h4>{cset.conceptSetNameOMOP} v{cset.version}</h4>
-        <List>
-          {Object.values(cset.concepts).map((concept, i) => {
-            return <ListItem style={{
-              margin: '3px 3px 3px 3px',
-              background: '#dbdbdb',
-              'border-radius': '5px',
-              'font-size': '0.8em'
-            }} key={i}>
-              {concept.conceptId}: {concept.conceptName}
-            </ListItem>
-          })}
-        </List>
-      </div>)
-  )
-}
+// todo: move this back in, or accept cols as params
+// https://www.ag-grid.com/react-data-grid/column-definitions/
+const getColumnDefs = () => {
+  return [
+    { field: 'ConceptID' },
+    // { field: '23007370' },
+    // { field: '23600781' },
+  ];
+};
 
-function ConceptList(props) {
-  // http://127.0.0.1:8000/fields-from-objlist?objtype=OmopConceptSetVersionItem&filter=codesetId:822173787|74555844
+// TODO: Page state refresh should not be 'on window focus', but on autocomplete widget selection
+// TODO: Should we move comparison logic python side and do query at new backend_url?
+function CsetComparisonPage(props) {
   const [qsParams, setQsParams] = useGlobalState('qsParams');
-  //const [filteredData, setFilteredData] = useState([]);
-  let codesetIds = qsParams && qsParams.codesetId && qsParams.codesetId.sort() || []
-
-  let url = backend_url('fields-from-objlist?') +
-      [
-        'objtype=OmopConceptSetVersionItem',
-        'filter=codesetId:' + codesetIds.join('|')
-      ].join('&')
+  let codesetIds = (qsParams && qsParams.codesetId && qsParams.codesetId.sort()) || []
+  let url = backend_url('concept-sets-with-concepts?concept_field_filter=conceptId&concept_field_filter=conceptName&codeset_id=' + codesetIds.join('|'))
 
   const { isLoading, error, data, isFetching } = useQuery([url], () => {
-    //if (codesetIds.length) {
-    //   console.log('fetching backend_url', url)
-      return axios.get(url).then((res) => res.data)
-      // console.log('enclave_url', enclave_url('objects/OMOPConceptSet'))
-      // .then((res) => res.data.data.map(d => d.properties))
-    //} else {
-      //return {isLoading: false, error: null, data: [], isFetching: false}
-    //}
-  });
-  // console.log('rowData', data)
-  return  <div>
-            <h4>Concepts:</h4>
-            <Table rowData={data} />
-          </div>
-  /*
-  let params = useParams();
-  let {conceptId} = params;
-  let path = `objects/OMOPConceptSet/${conceptId}/links/omopconcepts`;
-  let url = enclave_url(path)
-  const { isLoading, error, data, isFetching } = useQuery([path], () =>
-      axios
-          .get(url)
-          .then((res) => res.data.data.map(d => d.properties)) )
-  */
-}
+    return axios.get(url).then((res) => {
```

Comment:
FYI: logic for getting a data structure that shows concept set overlap / inclusion. Looks like:
`[{ConceptID: <conceptId>, <csetId1>: <'O' or 'X'>, ... <csetIdN>: <'O' or 'X'>}]`